README.md

```diff
@@ -18,19 +19,31 @@ Inference model integration SDK
 
 ## Integrating the SDK
 
-You should use this SDK to allow the Arterys web app to invoke your model. These sections will help you do so.
+You should use this SDK to allow the Arterys web app to invoke your model. 
+The `gateway.py` is a helper class that creates a Flask server to communicate with the Arterys app via HTTP.
+You will have to provide 2 endpoints:
+
+* `GET /healthcheck`: to tell whether the server is ready to handle requests
+* `POST /`: to handle inference requests
+
+### The healthcheck endpoint
```

Comment:
Let's mention explicitly that the Arterys app relies on the result from this endpoint to decide whether or not the inference service is ready to field requests. Returning READY too early will result in failures due to requests being sent too early.